dicoogle/src/main/resources/webapp/js/components/search/searchResult.jsx

```diff
@@ -326,8 +327,10 @@ const Step = React.createClass({
     else if (step > current) return "col-xs-3 wizardbar-item disabled";
     else if (step < current) return "col-xs-3 wizardbar-item completed";
   },
-  onStepClicked: function(current) {
-    this.props.onClick(current);
+  onStepClicked: function(newStep) {
+    if (this.props.current > newStep) {
```

Comment:
There's a regression here: in the previous version, you could enter a more specific step (e.g. Series) if you had already selected something at that level.

An example workflow:

1. Select a patient,
2. Select a study,
3. Go back to the patient list,
4. Select the Study step.

This would still work, but not anymore with these changes.